### ISSUE TITLE
XML Highlighter fails when using spaces after/before the equal sign in 

### ISSUE THREAD

**Issue by aeschli** [atom/language-xml#81]
_From @jens1o on May 12, 2017 19:30_

![image](https://cloud.githubusercontent.com/assets/11234139/26013784/35cd265c-375a-11e7-96b8-e566a10c5c5d.png)

```xml
<?xml version="1.0" encoding="UTF-8"?>

<!-- http://www.phpunit.de/manual/current/en/appendixes.configuration.html -->
<phpunit
    backupGlobals               = "false"
    backupStaticAttributes      = "false"
    colors                      = "true"
    convertErrorsToExceptions   = "true"
    convertNoticesToExceptions  = "true"
    convertWarningsToExceptions = "true"
    processIsolation            = "false"
    stopOnFailure               = "false"
    syntaxCheck                 = "false"
    bootstrap                   = "scss.inc.php">

    <testsuites>
        <testsuite name="Project Test Suite">
            <directory>tests</directory>
        </testsuite>
    </testsuites>

    <filter>
        <whitelist>
            <directory>src</directory>
        </whitelist>
    </filter>

</phpunit>

```

_Copied from original issue: Microsoft/vscode#26570_

**Comment by 50Wliu** [comment]
Fixed by #82.

**Comment by jens1o** [comment]
Thanks!
